lib/osm_community_index.rb

```diff
@@ -0,0 +1,26 @@
+module OsmCommunityIndex
+  def self.add_to_i18n
+    communities = Community.all
+    files = Dir.glob(Rails.root.join("node_modules/osm-community-index/i18n/*"))
```

Comment:
I would make that match `*.yaml` as an extra protection.

Reply:
Done.